styles/components.css

```diff
@@ -0,0 +1,85 @@
+.side-nav {
+    font-size: 1.4rem;
+    list-style: none;
+    margin-top: 3.5rem;
+
+
+}
+
+.side-nav__item {
+    position: relative;
+
+    &:not(:last-child) {
```

Comment:
please explain

Reply:
"&" is the name of the class. :not psuedo says "do not" and ":last-child" is the last child inside this class brackets.
In a easier way to think about it: Whatsever inside this class, do not do "this inside our css bracket" do the last child.